sdk/Lusid.Sdk.Tests/Utilities/TutorialBase.cs

```diff
@@ -145,24 +145,24 @@ private void UpsertRecipe(ConfigurationRecipe recipe)
                 .Select(inst => inst.Value.LusidInstrumentId)
                 .ToList();
             
-            // ADD instruments to the portfolio via their LusidInstrumentId
-            var transactionRequest = TestDataUtilities.BuildTransactionRequest(luids, scope, portfolioCode, effectiveFrom);
```

Comment:
At some point we should consider trying to break this function up. It's getting intense to read

Reply:
Yeah, this big method would probably get deleted in the future. This whole restructuring/refactoring replicates this method. But we are not quite there yet.